reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md

```diff
@@ -9,155 +9,299 @@ title:  Get-EventLog
 ---
 
 # Get-EventLog
+
 ## SYNOPSIS
-Gets the events in an event log, or a list of the event logs, on the local or remote computers.
+Gets the events in an event log, or a list of the event logs, on the local computer or remote
+computers.
+
 ## SYNTAX
 
 ### LogName (Default)
+
 ```
-Get-EventLog [-LogName] <String> [-ComputerName <String[]>] [-Newest <Int32>] [-After <DateTime>]
- [-Before <DateTime>] [-UserName <String[]>] [[-InstanceId] <Int64[]>] [-Index <Int32[]>]
- [-EntryType <String[]>] [-Source <String[]>] [-Message <String>] [-AsBaseObject] [<CommonParameters>]
+Get-EventLog [-LogName] <string> [[-InstanceId] <long[]>] [-ComputerName <string[]>] [-Newest
+<int>] [-After <datetime>] [-Before <datetime>] [-UserName <string[]>] [-Index <int[]>] [-EntryType
+<string[]>] [-Source <string[]>] [-Message <string>] [-AsBaseObject] [<CommonParameters>]
 ```
 
 ### List
+
 ```
-Get-EventLog [-ComputerName <String[]>] [-List] [-AsString] [<CommonParameters>]
+Get-EventLog [-ComputerName <string[]>] [-List] [-AsString] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The Get-EventLog cmdlet gets events and event logs on the local and remote computers.
 
-Use the parameters of Get-EventLog to search for events by using their property values.
-Get-EventLog gets only the events that match all of the specified property values.
+The **Get-EventLog** cmdlet gets events and event logs on the local and remote computers.
```

Comment:
\*\*Get-EventLog\*\*  should be \`Get-EventLog\`.

Reply:
Changed in all versions.